pkg/cloud/services/ec2/instances.go

```diff
@@ -622,6 +622,8 @@ func (s *Service) attachSecurityGroupsToNetworkInterface(groups []string, interf
 		return nil
 	}
 
+	s.scope.V(3).Info("updating security groups", "groups", totalGroups)
```

Comment:
Is V3 appropriate or would it be worthwhile to log this all the time?

Reply:
there are 2 other relevant logging messages that wrap this, but not with this much detail. i'm open to either.

Reply:
I'd probably err on the side of always logging, and do a follow-up to increase the V level if we find it too verbose when it's always on

Reply:
i think always would be better